csharp/src/Ice/RemoteException.cs

```diff
@@ -133,4 +149,51 @@ public static class RemoteExceptionExtensions
         /// <returns>A SlicedData value that provides the list of sliced-off slices.</returns>
         public static SlicedData? GetSlicedData(this RemoteException ex) => ex.SlicedData;
     }
+
+    public partial class ObjectNotExistException
+    {
+        /// <inheritdoc/>
+        protected override string DefaultMessage =>
+            $"could not find servant for Ice object `{Origin!.Value.Identity}'" +
```

Comment:
This does not look correct. The following should not crash:
```
var ex = new ObjectNotExistException();
Console.WriteLine(ex.Message);

```

Reply:
reworked the overrides to return `null` if Origin is not set